docs/user-guide.md

```diff
@@ -0,0 +1,187 @@
+
+To stop your application, you can either right-click on the application listed in the Liberty tool window, and click on the `Stop` action or select the `Liberty: Stop` action in the "Go to Action" IntelliJ IDEA menu.
+
+![Liberty stop action](images/liberty-stop-action.png)
+
+## Debugging your application
+
+To debug your application, you can start dev mode with the debugger automatically attached to the Liberty server JVM running your application.
+
+To start dev mode with the debugger attached, first create or select a Liberty Run/Debug Configuration through the IntelliJ Run/Debug Configuration menu. Once you select the Liberty Run/Debug Configuration, select the `Debug` action next to the Run/Debug Configuration menu.
+
+A new terminal tab will open to the run the application in dev mode and the debugger will wait to attach.
+
+![Liberty debug action](images/liberty-debug.png)
+
+The debug dialog will pop up to confirm your Debug Configuration. Select "Debug".
+
+![Liberty debug confirm configuration](images/liberty-debug-confirm-config.png)
+
+Once the server starts, the IntelliJ IDEA debugger will attach and switch to the debug perspective. You can now set breakpoints and debug your code as usual.
+
+![Liberty debugger attached](images/liberty-debugger-attached.png)
+
+
+## Manually adding your Liberty project to the tool window
+
+To manually add your Liberty project to the Liberty tool window, select the `Liberty: Add project to the tool window` action in the "Go to Action" IntelliJ IDEA menu. 
+
+![Liberty add project to tool window](images/liberty-add-project-to-tool-window.png)
+
+You will be prompted with a list of projects that are not already displayed in the Liberty tool window.
+
+![Liberty add project to tool window selection](images/liberty-add-project-to-tool-window-selection.png)
+
+You can remove manually added Liberty projects from the Liberty tool window by selecting the `Liberty: Remove project from the tool window` action in the "Go to Action" IntelliJ IDEA menu.
+
+## Configuring a Liberty server
+
+Liberty configuration assistance is offered through the Liberty Config Language Server. For more information, see the [project documentation in GitHub](https://github.com/OpenLiberty/liberty-language-server#liberty-config-language-server).
+
+1. Start the project in dev mode, using one of the Liberty tool window start commands above. This will install the Liberty features required for your application and allow generation of a corresponding server.xml schema file.
+2. Open your server.xml file. Proceed to use Liberty specific editing support: completion by typing "Ctl/Cmd + Space" at a given point within the document.
+3. Open your bootstrap.properties or server.env file. Proceed to use Liberty specific editing support: completion by typing "Ctl/Cmd + Space" at a given point within the document.
+
+![LCLS server.xml completion](images/LCLS-server-xml-completion.png)
+
+## Developing with Jakarta EE and MicroProfile APIs
+
```

Comment:
suggest revising this section in a similar fashion to the previous section suggestions